src/test/java/org/openrewrite/featureflags/launchdarkly/UpgradeLaunchDarkly7Test.java

```diff
@@ -69,7 +69,7 @@ void mavenDependency() {
                   </project>
                   """,
                 spec -> spec.after(actual -> {
-                      Matcher matcher = Pattern.compile("<version>(7\\.\\d\\.\\d+)</version>").matcher(actual);
+                      Matcher matcher = Pattern.compile("<version>(7[.][0-9]+[.][0-9])</version>").matcher(actual);
```

Comment:
So the regex here that is the issue was position 2 expected a single digit while for the Gradle one it allowed 1+. It seems like a typo was the issue and all that would have been needed was to add a + for the middle digit selector.

The new regex also suffers from the same issue, but now if the patch version were to become two digits instead. (But both are consistent now too).

nit: the notation used here for the literal dots is kinda weird IMO.